src/create_string.cpp

```diff
@@ -3,12 +3,18 @@
 #include <string.h>
 #include "create_string.h"
 
-char* create_string(Local<Value> value) {
-  if (value->IsNull() || !value->IsString()) {
+char* create_string(Nan::MaybeLocal<v8::Value> maybevalue) {
+  v8::Local<v8::Value> value;
+  
+  if (maybevalue.ToLocal(&value)) {
+    if (value->IsNull() || !value->IsString()) {
+      return 0;
+    }
+  } else {
     return 0;
```

Comment:
I'll revisit it later - we might need to review `create_string()` anyway to better comply with libsass API. 

For now I have more issues to worry than V8 sending me wrong UTF-8 :)